README.md

```diff
@@ -58,6 +58,8 @@ track, the second, that a face is no longer visible.
 
 TODO
 ====
-1. Make it work with Kinect.
-2. Migrate to Python cv2 wrappers from cv.  The cv wrappers use a
-   c++-like interface, the cv2 wrappers use numpy and are more efficient.
+1. Implement face recognition on a "new_face"
```

Comment:
In principle ("in theory", "in the long term"), such recognition is supposed to be done in the AtomSpace, and not here, as that is the place where all of the different data streams come together. For example, perhaps the voice can be recognized as belonging to so-n-so, this can then be correlated with the face that is being viewed. 

In practice, we don't actually have voice recognition. Nor do we have a smile detector,  A laugh detector. A shyness detector. etc. etc. I have some sketches on how to handle generic sensory data but most of it remains rather futuristic.  Anyway, this is a discussion for .. somewhere other than here.